packages/react-scripts/config/loaders/getAssetLoaders.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
This code is identical in two configs. Extracting helps to avoid undesired mismatch